app/models/miq_worker.rb

```diff
@@ -286,12 +286,8 @@ def queue_name
     end
   end
 
-  def self.supports_container?
-    false
-  end
-
   def self.containerized_worker?
-    MiqEnvironment::Command.is_podified? && supports_container?
+    MiqEnvironment::Command.is_podified?
```

Comment:
10 lines below there is also a supports_systemd?  Should we remove that one as well for consistency?

Reply:
That one checks Settings so you can disable it if something isn't working right and also make sure the systemd gem is installed so its not just a static true/false like `supports_container?` is.